invenio_config/__init__.py

```diff
@@ -22,19 +22,148 @@
 # waive the privileges and immunities granted to it by virtue of its status
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
-"""Invenio configuration module."""
+"""Invenio configuration module.
```

Comment:
"Invenio configuration loader" to avoid the term "module"?  (see also `setup.py` where we need to amend docstring.)